docs/standard/glossary.md

```diff
@@ -0,0 +1,265 @@
+
+UWP was the first application framework supported by .NET Native. Now, we support building native console apps for Windows, macOS, and Linux.
+
+See [Intro to .NET Native and CoreRT](https://github.com/dotnet/corert/blob/master/Documentation/intro-to-corert.md)
+
+## .NET Standard
+
+A formal specification of .NET APIs that are available in each .NET implementation.
+
+See [.NET Standard](net-standard.md).
+
+## NGEN
+
+Native (image) generation.
+
+You can think of this technology as a persistent JIT compiler. It usually compiles code on the machine where the code is executed, but compilation typically occurs at install time.
+
+## package
+
+For a package-based target framework, a NuGet package that contains an assembly of the same name.
+
+The package is a *.zip* file wih  a *.nupkg* extension that may contain assets (such as *.dll* files and *.xml* files) for use with multiple frameworks and versions. When installed in an app or library, the appropriate assets are selected based on the target framework specified by the app or library. The assets that define the interface are in the *ref* folder, and the assets that define the implementation are in the *lib* folder.
+
+## platform
+
+An operating system and the hardware it runs on, such as Windows, macOS, Linux, iOS, and Android.
+
+Following are examples of usage in sentences:
+
+- ".NET Core is a cross-platform implementation of .NET." 
+- "PCL profiles represent Microsoft platforms, while the .NET Standard is agnostic to platform."
+
+The .NET documentation frequently uses ".NET platform" to mean either an implementation of .NET or the .NET stack including all implementations. Both of these usages tend to get confused with the primary (OS/hardware) meaning, so we plan to eliminate these usages from the documentation.
+
+## runtime
+
+The execution environment for a managed program.
+
+The OS is part of the runtime environment but is not part of the .NET runtime. Here are some examples of .NET runtimes:
+
+- Common Language Runtime (CLR)
+- Core Common Language Runtime (CoreCLR)
+- .NET Native (for UWP)
+- Mono runtime
+
+The .NET documentation frequently uses "runtime" to mean an implementation of .NET. For example, in the following sentences "runtime" should be replaced with "implementation":
+
+- "The various .NET runtimes implement specific versions of .NET Standard."
+- "Libraries that are intended to run on multiple runtimes should target this framework." (referring to .NET Standard)
+- "The various .NET runtimes implement specific versions of .NET Standard. … Each .NET runtime version advertises the highest .NET Standard version it supports …"
+
+We plan to eliminate this inconsistent usage. 
+
+## stack
+
+A set of programming technologies that are used together to build and run applications.
+
+"The .NET stack" refers to the .NET Standard and all .NET implementations. The phrase "a .NET stack" may refer to one implementation of .NET. 
+
+## target framework
```

Comment:
target framework also includes a version. So a target framework is a versioned .NET implementation.